lib/bap/bap.mli

```diff
@@ -950,6 +950,38 @@ module Std : sig
     module Make(Interval : Interval) : S
       with type key := Interval.t
        and type point := Interval.point
+
+    (**  Binable Abstract Interval.
+
+         An extension of the Interval signature that supports the
+         necessary extensions to be serializable as a
+         {{!Value}universal value}.
```

Comment:
Well, in fact it has nothing to do with Values, so the comment is rather confusing. Can you fix it to "An extension of the Interval signature with the Binable interface"